cabal.project.local~

```diff
@@ -0,0 +1 @@
+tests: True
```

Comment:
Maybe this wasn't intended to be committed?

Reply:
oops